checker/tests/index/MinLenFromPositive.java

```diff
@@ -0,0 +1,21 @@
+import org.checkerframework.checker.index.qual.Positive;
+import org.checkerframework.common.value.qual.*;
+
+class MinLenFromPositive {
+
+    void test(@Positive int x) {
+        int @MinLen(1) [] y = new int[x];
+        @IntRange(from = 1) int z = x;
+        @Positive int q = x;
+    }
+
+    @SuppressWarnings("index")
```

Comment:
Unfortunately, no.  The framework directory is not suppose to have any dependency on the checker directory.